### DISCUSSION TITLE
Non-coding tasks

### DISCUSSION THREAD

**Discussion by mitko** [mitko/readable_climate_reports#9] [General]
Starting a discussion to welcome @marcheaterre and discuss non-coding tasks which can support this project. 

Here are some ideas, but feel free to suggest anything you think is missing and plays well to your strengths:
* Investigate rendering formats - how to best present the reports, so that they are more accessible.
* Research similar projects that we can learn from and not duplicate work. There probably are many projects which try to make scientific papers more readable. Can we get some partnerships with them?
* gain a deeper understanding of the content of the latest IPCC reports - what are the most important parts of content, and how do we make sure an uninformed reader will be able to quickly discover and grok them ?
* Where do we find the data behind the report?
....

**Comment by petermr** [comment]
## Create dictionary of terms, abbreviations, acronyms

For the interested semi-informed reader (I'm one) busting the jargon is critical:
* first step - identify the jargon
* add interpretation
  - from online authorities (I can offer semi-automatic lookup of Wikipedia/Wikidata)
  - internal glossaries if they exist
  - human annotation
 

We have a tested approach to creating multiple dictionaries based on Wikidata. We are also using Python tools (NLTK, Sci/SpaCy) for NLP and information extraction
